core/domain/exp_services_test.py

```diff
@@ -2945,6 +3250,20 @@ def test_get_lower_bound_wilson_rating_from_exp_summary(self):
             exp_services.get_scaled_average_rating(exp.ratings),
             2.056191454757, places=4)
 
+    def test_valid_demo_file_path(self):
```

Comment:
I'm not sure whether this should be a part of the exp_service test, is it covering any method in exploration?

Reply:
I just think this is good measure which checks for invalid demo file paths. Do you want it removed?